src/Control/Monad/Eff/JQuery.purs

```diff
@@ -1,12 +1,67 @@
 -- | This module defines foreign types and functions for working with
 -- | the jQuery library.
 
-module Control.Monad.Eff.JQuery where
+module Control.Monad.Eff.JQuery
```

Comment:
I exported everything it was already exported. However I would like you to review the change, just in case I forgot to export something.
The idea here is to not export the getAttrImpl, it is internal.